index.d.ts

```diff
@@ -18,6 +18,8 @@ declare module 'preact' {
 declare module 'enzyme' {
   export type NodeType = 'function' | 'class' | 'host';
 
+  export type RSTNodeTypes = Exclude<ComponentChild, VNode> | RSTNode;
```

Comment:
Perhaps name this `RSTNodeChild` to indicate the relation with `ComponentChild`?